KJ1124/Thor_Puzzle_assign.c

```diff
@@ -0,0 +1,118 @@
+// *******************
+// **Power of Thor**
+// *******************
+/*
+Use `this link <https://www.codingame.com/ide/puzzle/power-of-thor-episode-1>`_ to access the coding game.
+
+
+General Instructions
+--------------------
+
+**Objective**
+
+Write a program that will guide Thor to reach the light of power in the most efficient manner.
+
+**Game Rules**
+
+Thor will move on this map that has the coordinates of 40 wide by 18 high. As the light of power changes locations your goal is change the direction of Thor's movements based on the coordinate placement of the light of power.
+
+
+*Note: You can utilize any coding language that you are most comfortable with to write this code by changing this option in the code tab. I chose to use C++*
+
+*/
+
+#include <iostream>
+#include <string>
+#include <vector>
+#include <algorithm>
+
+using namespace std;
+/*
+Provided Code
+--------------
+*/
+/* These are the functions and variables that are also provided to the developer when you begin the coding game.
+*/
+int main() 
+// beginning bracket
+{
+    // the X coordinate position of the light of power
+    int light_x;
+    // the Y coordinate position of the light of power
+    int light_y; 
+    // Thor's starting position at X
+    int initial_tx; 
+    // Thor's starting position at Y
+    int initial_ty;
+    // The cin.ignore() function is used to ignore characters that may not belong in your input buffer
+    cin >> light_x >> light_y >> initial_tx >> initial_ty; cin.ignore();
+
+/* This while statement is the beginning of the beginning of the game loop and it allows causes all of your test cases to run
+*/
+    while (1) {
+        
+        int remaining_turns; 
+        // This is the remaining amount of turns Thor can move. Do not remove this line.
+        cin >> remaining_turns; cin.ignore();
+
+/*
+Diagram
+---------
+
+.. image:: ThorPwr.jpg
+
+
+*/
```

Comment:
Combine these two comments -- there's no reason to end one then just begin another.